source/agora/node/Runner.d

```diff
@@ -144,6 +144,15 @@ public Listeners runNode (Config config)
         auto settings = new HTTPServerSettings(interface_.address);
         settings.port = interface_.port;
         settings.rejectConnectionPredicate = isBannedDg;
+        // Enable TLS
+        if (exists("agora-cert.pem") && exists("agora-key.pem"))
+        {
+            import vibe.stream.tls;
+            settings.tlsContext = createTLSContext(TLSContextKind.server);
+            settings.tlsContext.useCertificateChainFile("agora-cert.pem");
+            settings.tlsContext.usePrivateKeyFile("agora-key.pem");
+            log.info("TLS Enabled..");
+        }
```

Comment:
Should apply to admin interface as well.

Reply:
Also, it should probably just look up a few places for `cert` and `key`

Reply:
Updated